### PR TITLE
test serializability on all domain models

### PR DESCRIPTION
**Pull request by Adam-Hammo**
Quick serverside checking of SDK->domain conversions to catch edge cases that might cause serialising problems. Also see https://covidence.slack.com/archives/C0810ARPCRG/p1750382080141059.

### CODE REVIEW

File: app/domain/imports/models/models.py [hunk 1]
```diff
@@ -154,9 +154,12 @@ class ImportRecord(DomainBaseModel, SQLAttributeMixin):
     async def from_sdk(cls, data: destiny_sdk.imports.ImportRecordIn) -> Self:
```
Comment (danielbreves) [inline app/domain/imports/models/models.py:154]: I wonder if we could implement a mixin for these?

Comment (Adam-Hammo) [reply app/domain/imports/models/models.py:154]: Yeah I did think about that - an abstract mixin would be nice but pydantic doesn't play so nice with abstract classes. I think the actual function bodies, even though they're relatively similar at the moment, will possible diverge over time.
